lib/index.ts

```diff
@@ -0,0 +1,9 @@
+import Onyx from './Onyx';
+import type {OnyxUpdate, ConnectOptions} from './Onyx';
+import type {CustomTypeOptions, OnyxCollection, OnyxEntry, NullishDeep, KeyValueMapping, OnyxKey, Selector, WithOnyxInstanceState} from './types';
+import useOnyx from './useOnyx';
+import withOnyx from './withOnyx';
+
+export default Onyx;
+export {withOnyx, useOnyx};
+export type {CustomTypeOptions, OnyxCollection, OnyxEntry, OnyxUpdate, ConnectOptions, NullishDeep, KeyValueMapping, OnyxKey, Selector, WithOnyxInstanceState};
```

Comment:
```suggestion
export type {CustomTypeOptions, OnyxCollection, OnyxEntry, OnyxUpdate, ConnectOptions, NullishDeep, KeyValueMapping, OnyxKey, Selector, WithOnyxInstanceState, UseOnyxResult, OnyxValue, FetchStatus};
```

Let's export these two like before, they can be useful for the library consumers

Reply:
Will do in another PR, thanks!